app/jobs/story_update_job.rb

```diff
@@ -20,7 +20,7 @@ def receive_story_update(data)
     load_resources(data)
     episode ? update_episode : create_episode
```

Comment:
episodes require a podcast_id to be created, so we won't ever make an episode for a story when that story isn't both in a series, and that series already has a podcast previously created for it.

So if there is no series for the story at all, it won't get past `create_episode` where is checks for that.
If there is a series, but no matching podcast, it will fail on `Podcast.find_by!`.

If I handle that differently, and just return a nil episode,  `episode.copy_media` will fail with a no method error on nil object, I should probably make that a `try` to prevent that error bubbling up.